price_security/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+  :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+  :alt: License: AGPL-3
+
+==============
+Price Security
+==============
+
+Creates a new permission to restrict the users that can modify the prices
+of the products.
+
+Asociate to each user a list of pricelist and the correspoding discounts they
+can apply to sale orders and invoices.
+
+Allow the posibility to mark products so that anyone can modify their price in
+a sale order.
+
+
+NEW: sequence field on pricelist and payment term (you can only assigne pricelests or terms of lower priority than partner default one)
```

Comment:
Sacar un enter (hay dos con el párrafo anterior). Sacar el new, que quede (tmb corregí terminos assign y pricelist):
Add a sequence field on pricelist and payment term (you can only assign pricelist or terms of lower priority than partner default one)